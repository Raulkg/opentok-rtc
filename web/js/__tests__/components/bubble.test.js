var View = require('../../components/bubble.js');
test('snapshot for bubble component', () => {
  expect(View).toMatchSnapshot();
});