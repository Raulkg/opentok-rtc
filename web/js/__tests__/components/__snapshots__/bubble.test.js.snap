// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot for bubble component 1`] = `
Object {
  "BubbleFactory": Object {
    "get": [Function],
  },
}
`;
